volkswagencarnet/vw_connection.py

```diff
@@ -106,29 +107,27 @@ async def doLogin(self, tries: int = 1):
         self._session_logged_in = True
 
         # Get VW-Group API tokens
-        if not await self._getAPITokens():
-            self._session_logged_in = False
-            return False
+        # if not await self._getAPITokens():
+        #    self._session_logged_in = False
+        #    return False
 
         # Get list of vehicles from account
         _LOGGER.debug("Fetching vehicles associated with account")
-        await self.set_token("vwg")
+        # await self.set_token("vwg")
         self._session_headers.pop("Content-Type", None)
-        loaded_vehicles = await self.get(
-            url=f"https://msg.volkswagen.de/fs-car/usermanagement/users/v1/{BRAND}/{self._session_country}/vehicles"
-        )
+        loaded_vehicles = await self.get(url=f"{BASE_API}/vehicle/v2/vehicles")
         # Add Vehicle class object for all VIN-numbers from account
-        if loaded_vehicles.get("userVehicles") is not None:
+        if loaded_vehicles.get("data") is not None:
             _LOGGER.debug("Found vehicle(s) associated with account.")
-            for vehicle in loaded_vehicles.get("userVehicles").get("vehicle"):
-                self._vehicles.append(Vehicle(self, vehicle))
+            for vehicle in loaded_vehicles.get("data"):
+                self._vehicles.append(Vehicle(self, vehicle.get("vin")))
         else:
             _LOGGER.warning("Failed to login to We Connect API.")
             self._session_logged_in = False
             return False
 
         # Update all vehicles data before returning
-        await self.set_token("vwg")
+        # await self.set_token("vwg")
```

Comment:
There are many commented out lines, like this one. If the code is no longer needed, it would be best removing it.

Reply:
Yes, I plan to do that once I’m very sure I won’t need it again.
There are several clean-up jobs I keep in the back of my head 😅
I just assumed sharing is a higher priority currently than cleaning up

Reply:
Indeed, great job on being so reactive and proactive on the issue, well appreciated 👍